service/component-samples/rv/src/main/java/org/fido/iot/sample/RvContextListener.java

```diff
@@ -45,6 +45,18 @@ public void contextInitialized(ServletContextEvent sce) {
     ds.setMaxOpenPreparedStatements(100);
 
     final CryptoService cs = new CryptoService();
+    String epidTestMode = sc.getInitParameter(RvAppSettings.EPID_TEST_MODE);
```

Comment:
So, based on this check, the default behavior of epid_test_mode would be false. Is that correct?

Reply:
Yes